python/tests/test_static_disk_index.py

```diff
@@ -36,8 +36,8 @@ class TestStaticDiskIndex(unittest.TestCase):
     def setUpClass(cls) -> None:
         cls._test_matrix = [
             _build_random_vectors_and_index(np.float32, "l2"),
-            _build_random_vectors_and_index(np.uint8, "l2"),
-            _build_random_vectors_and_index(np.int8, "l2"),
+            # _build_random_vectors_and_index(np.uint8, "l2"),
+            # _build_random_vectors_and_index(np.int8, "l2"),
```

Comment:
Should we just remove these two lines? Is there a plan to bring them back?

Reply:
ach no I commented them out to speed up my test feedback loop.  I also need to add the cosine and mips metrics too